src/commcare_cloud/ansible/roles/pg_backup/templates/plain/create_postgres_dump.sh.j2

```diff
@@ -46,12 +57,16 @@ if [[ $? -ne 0 ]] ; then
     echo "Postgres Backup : Tar Failed"
     exit 2
 else
+    if [[ postgres_backup_hourly && "$BACKUP_TYPE" == "hourly" ]]; then
+        find {{ postgresql_backup_dir }} -mmin "+${HOURS_TO_RETAIN_BACKUPS}" -name "postgres_${ENV}_${BACKUP_TYPE}_*" ! -path "*weekly*" ! -path "*daily*" -delete;
+        find {{ postgresql_backup_dir }} -mmin "+${HOURS_TO_RETAIN_BACKUPS}" -name "postgres_${BACKUP_TYPE}_*" ! -path "*weekly*" ! -path "*daily*" -delete;
```

Comment:
This filename format hasn't existed for hourly backups, so I don't think this line is necessary